### PR TITLE
chore(accessibility): remove `defaultBehavior`

### PR DESCRIPTION
**Pull request by layershifter**
# BREAKING CHANGES

This PR removes `defaultBehavior` because the single reason to have it is how `getAccessibility()` function is working.

### CODE REVIEW

File: packages/react/src/lib/renderComponent.tsx [hunk 1]
```diff
@@ -53,18 +57,25 @@ export interface RenderConfig<P> {
 }
 
 const getAccessibility = (
-  props: State & PropsWithVarsAndStyles,
+  props: State & PropsWithVarsAndStyles & { accessibility?: Accessibility },
   actionHandlers: AccessibilityActionHandlers,
   isRtlEnabled: boolean,
-) => {
-  const { accessibility: customAccessibility, defaultAccessibility } = props
```
Comment (layershifter) [inline packages/react/src/lib/renderComponent.tsx:60]: We don't have `defaultAccessibility` prop at all 💥

**Review by jurokapsiar — APPROVED** [review]

File: docs/src/prototypes/MenuButton/MenuButton.tsx [hunk 2]
```diff
@@ -150,10 +150,11 @@ class MenuButton extends React.Component<MenuButtonProps, MenuButtonState> {
     const { button, disabled, menu, placement } = this.props
     const { menuOpen } = this.state
     const [position, align] = _.split(placement, '-') as [Position, Alignment]
-    const accessibilityBehavior: ReactAccessibilityBehavior = menuButtonBehavior({
-      ...this.props,
-      ...this.state,
-    })
+    const accessibilityBehavior: ReactAccessibilityBehavior = {
+      attributes: {},
```
Comment (sophieH29) [inline docs/src/prototypes/MenuButton/MenuButton.tsx]: why do you need an empty `attributes` object here? I see attributes already exist in `menuButtonBehavior`? and also `keyHandlers` which aren't used. I mean I understand why you're adding these empty mocks to replicate same behavior as in `renderComponent`, but I think it's not very understandable for someone who is looking at this prototype. And I believe if you create a behavior which returns an object people are wise on which object's props they can reference to

**Review by sophieH29 — APPROVED** [review]
small comment! Other than that, great job 👍 ❤️

**Review by bmdalex — APPROVED** [review]

File: docs/src/prototypes/MenuButton/MenuButton.tsx (hunk 2 above)
Comment (layershifter) [reply docs/src/prototypes/MenuButton/MenuButton.tsx]: Extracted this to empty behavior

**Pull request merged**